internal/v2/s2av2.go

```diff
@@ -57,12 +68,35 @@ func (c *s2av2TransportCreds) ClientHandshake(ctx context.Context, serverAuthori
 	if err != nil {
 		serverName = serverAuthority
 	}
-	// TODO(rmehta19): Create a stub to S2Av2.
+	// Create a stream to S2Av2.
+	opts := []grpc.DialOption {
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+		grpc.WithReturnConnectionError(),
+		grpc.WithBlock(),
+	}
+	conn, err := grpc.Dial(*fakes2av2Address, opts...)
+	if err != nil {
+		log.Fatalf("Client: failed to connect: %v", err)
+	}
+	defer conn.Close()
+	client := s2av2pb.NewS2AServiceClient(conn)
+	log.Printf("Client: connected to: %s", *fakes2av2Address)
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+
+	// Setup bidrectional streaming session.
+	callOpts := []grpc.CallOption{}
```

Comment:
nit: I think we can omit the local variable and just directly pass []grpc.CallOption{} to the SetUpSession method.